src/js/utils.js

```diff
@@ -1,6 +1,7 @@
-import {API_KEY, API_URL, IMG_URL} from "./constants";
+import {API_KEY, API_URL, IMG_URL, SEARCH_URL} from "./constants";
 
 export const generateApiUrl = path => `${API_URL}${path}?api_key=${API_KEY}`
 export const generateImageUrl = path => `${IMG_URL}${path}`
+export const generateSearchUrl = path => `${SEARCH_URL}?api_key=${API_KEY}&language=en-US&query=${path}&page=1&include_adult=false`
```

Comment:
Needless props, you can write shortest - `${SEARCH_URL}?api_key=${API_KEY}&query=${path}`